example/build.gradle

```diff
@@ -0,0 +1,64 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    namespace 'org.xmtp.android.example'
+    compileSdk 33
+
+    defaultConfig {
+        applicationId "org.xmtp.android.example"
+        minSdk 28
```

Comment:
Any reason for 28 min? Looks like it's 77% of adoption which is _p_ good but dropping down to 27 seems to bring coverage up to 85% which is a p big jump. No worries if this is necessary tho!

<img width="678" alt="Screenshot 2022-12-21 at 3 12 22 PM" src="https://user-images.githubusercontent.com/556051/208994833-e4ceb18a-de11-44c3-8aa3-8f4e552f663c.png">
<img width="428" alt="Screenshot 2022-12-21 at 3 11 17 PM" src="https://user-images.githubusercontent.com/556051/208994835-19c64d1a-d537-4d8d-bb8d-8beb5fb14468.png">

>Screenshots are from Android Studio when creating a new project and selecting the OS.

Reply:
I have a memory about compose or material3 not being supported great all the way back but we can re-adjust that whenever we come across it. I can drop to 27 for now. 👍

Reply:
We could maybe even go lower on our SDK down to 26 and higher for our example app since mostly UI problems on the lower versions.

Reply:
26 and 27 didn't seem to make as much of a difference fwiw! (It was like 1%) But agree w the approach.